feature-utils/poly-analysis/src/report/report-analysis.js

```diff
@@ -0,0 +1,14 @@
+import RootAnalysis from "../../root-analysis.js";
+
+export default class ReportAnalysis extends RootAnalysis {
```

Comment:
... and also might cause repetition of code. This is exactly the same code as above in `feature-utils/poly-analysis/src/analysis/report/report-analysis.js`

Reply:
yep will remove it from the facebookImport module as it is probably going to be shared between features and their analyses 👍